libr/anal/p/anal_arm_cs.c

```diff
@@ -1022,53 +1022,51 @@ static int analop64_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int l
 		default:
 		    break;
 		}
-		if ((int)MEMDISP64(1) < 0) {
```

Comment:
I couldn't understand this if: why having a negative offset should result in an direct ESIL translation, while for positive offset you still have to check other things - see below?

Also, resulting expression is something like `dst,base,20,-,=[8]` which to my understanding translates to `dst = *(20-base)`